README.md

```diff
@@ -23,8 +19,8 @@ Wow, well, loads. Here is a quick list.
 * Loads of code removed for a faster, more reliable PhotoShow. Hopefully.
 * Uploaders have lost a load of rights. Basically, all they can do is upload. No file removing anymore.
 * Rights management SHOULD be the same as before. But, just in case, please check after updating
-* Updated phpThumb library
-* Added configuration values for `quality_small` and `quality_mini` thumbnails with default of `90` - change them through your own config.php, like `$config->quality_small = 90;`  
+* Updated phpThumb library

```

Comment:
Why this other documentation change?

Reply:
This Happyend accidentaly. I used the github Web editor :/